code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -94,26 +94,27 @@
 				new cat_type(loc)
 
 
-/mob/living/simple_animal/pet/cat/handle_automated_action()
+/mob/living/simple_animal/pet/cat/Life()
 	..()
-	if(prob(1))
-		custom_emote(1, pick("stretches out for a belly rub.", "wags its tail.", "lies down."))
-		icon_state = "[icon_living]_rest"
-		resting = 1
-		update_canmove()
-	else if (prob(1))
-		custom_emote(1, pick("sits down.", "crouches on its hind legs.", "looks alert."))
-		icon_state = "[icon_living]_sit"
-		resting = 1
-		update_canmove()
-	else if (prob(1))
-		if (resting)
-			custom_emote(1, pick("gets up and meows.", "walks around.", "stops resting."))
-			icon_state = "[icon_living]"
-			resting = 0
+	make_babies()

```

Comment:
It seems weird to put this in Life() (which always runs) rather than say handle_automated_movement() (which gets turned off if there are no mobs on the zlevel).

Reply:
This is the standard practice for `make_babies` and is completely and utterly consistent across the codebase.